lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -1470,6 +1495,12 @@ instance ToSchema ApiTxId where
 instance ToSchema WalletPostData where
     declareNamedSchema _ = declareSchemaForDefinition "ApiWalletPostData"
 
+instance ToSchema AccountPostData where
+    declareNamedSchema _ = declareSchemaForDefinition "ApiAccountPostData"
+
+instance ToSchema WalletOrAccountPostData where
+    declareNamedSchema _ = declareSchemaForDefinition "ApiWalletOrAccountPostData"
```

Comment:
will need probably to do something here, utilize `proxy` in `declareNamedSchema proxy` and basing on that in one case 
```
declareSchemaForDefinition "ApiAccountPostData"
```
and 
```
declareSchemaForDefinition "ApiWalletPostData"
```

Reply:
fixed in 794224a